docker/hq-compose.yml

```diff
@@ -71,7 +71,7 @@ services:
     expose:
       - "5432"
     healthcheck:
-      test: pg_isready
+      test: pg_isready -d postgresql://commcarehq:commcarehq@postgres
```

Comment:
Thanks. I like your suggestion to use explicit params.

You're right, I used a connection string because I assumed that if you need to specify a user, you'd need to specify their password too. (The fact that that isn't true feels counter-intuitive. I'd like to believe they have reasons, but if they need a username, I would expect they'd try to authenticate it. I'd also think that having a separate param for connection string (`-c`?) would make sense. The database name is optional in a connection string, so passing it with the `dbname` param is, as you pointed out, not what I expected.)